### PR TITLE
EEC-7: Add app skeleton

### PR DESCRIPTION
**Pull request by jamiecarterHO**
## What?

[EEC-7](https://collaboration.homeoffice.gov.uk/jira/browse/EEC-7)

Adds the necessary files to create the eVisa - Error correction form skeleton. This includes the Node.js/HOF and Docker config files needed to build and run a HOF app, plus the basic folder structure for the EEC app itself with minimal content added.

Also includes housekeeping files such as README update, pull request template and ignore files for Docker and Git.

## Why?

This skeleton should allow a developer to run a local version of this app with either Node.js or docker-compose, and work on page specific tickets for the form.

## Anything else?

This PR does not include HOF devops setup such as the /bin, /kube folders or drone.yml. These can be appended to this feature or added as a new feature later. The purpose of this ticket is to enable local development of form pages.

### CODE REVIEW

**Review by dk4g — APPROVED** [review]
Just a few minor things due to redundant code. Once addressed, can be merged into master

File: apps/eec/translations/src/en/buttons.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "edit": "Change",
+  "continue": "Continue",
+  "start-now": "Report an error",
```
[the remark this answers is not in the document]
Comment (jamiecarterHO) [reply apps/eec/translations/src/en/buttons.json]: Updated

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "evisa-error-correction",
+  "version": "1.0.0",
+  "description": "eVisa error correction",
+  "main": "index.js",
+  "engines": {
+    "node": ">=20.16.0 <21.0.0"
```
[the remark this answers is not in the document]
Comment (jamiecarterHO) [reply package.json]: Updated

File: apps/eec/translations/src/en/pages.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "start": {
+    "header": "Report an error with your eVisa"
+  },
```
[the remark this answers is not in the document]
Comment (jamiecarterHO) [reply apps/eec/translations/src/en/pages.json]: Updated

**Pull request merged**